docs/src/main/sphinx/admin/properties-writer-scaling.rst

```diff
@@ -35,9 +35,20 @@ divided by the number of writers per task. This can be specified on a
 per-query basis using the ``writer_min_size`` session property.
 
 ``use-preferred-write-partitioning``
-^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 * **Type:** ``boolean``
 * **Default value:** ``false``
 
 Enables preferred write partitioning. This option will force every partition
 to be written by a single worker.
+
+``preferred-write-partitioning-min-number-of-partitions``
```

Comment:
follow up PR. I would like @mosabua to review it.

Reply:
Maybe a follow-up commit? Features and docs should IMO be added atomically.